File: LicentaUI/Models/ReviewSeriesModel.cs
namespace LicentaUI.Models
{
    public class ReviewSeriesModel
    {
        public string ID { get; set; }

        public string IdUser { get; set; }

        public string IdSeries { get; set; }

        public int? Grade { get; set; }

        public string Review { get; set; }

        public Status Status { get; set; }
    }
}

File: LicentaAngular/src/app/models/review-series-model.ts
import { Status } from './status';
import { Genre } from './genre';
import { Rating } from './rating';

export interface ReviewSeriesModel {
    idReview: string;
    idUser: string;
    username: string;
    idFilm: string;
    grade: number;
    review: string;
    status: Status;
    series: {
        title: string;
        director: string;
        description: string;
        relaseDate: Date;
        prequeslId: string;
        sequelId: string;
        genre: Genre;
        rating: Rating;
        nEp: number;
        picture: string;
    }
}
